spec/models/miq_server/environment_manager_spec.rb

```diff
@@ -3,14 +3,12 @@
 RSpec.describe "Server Environment Management" do
   context ".get_network_information" do
     let(:mac_address) { 'a:1:b:2:c:3:d:4' }
-    let(:hostname)    { "localhost" }
+    let(:hostname)    { `hostname -f`.chomp }
```

Comment:
@Fryguy the `miq_environment_spec` has the following skip:
```
example "fully_qualified_domain_name" do
  skip if RUBY_PLATFORM.include?("darwin")
  expect(described_class.fully_qualified_domain_name).to eq(`hostname -f`.chomp)
end
```

Do we need to do that here as well?  I think you mentioned that `hostname -f` works on mac

Reply:
This change now works on my machine, so this is fine for me